showCrime/requirements.dev.txt

```diff
@@ -0,0 +1,2 @@
+django-debug-toolbar
```

Comment:
Add versions to all requirements. Pylint especially has a history of releasing breaking changes.

Reply:
Sure thing!